easybuild/easyconfigs/a/azure-cli/azure-cli-2.47.0-GCCcore-10.3.0-Python-3.9.5.eb

```
@@ -0,0 +1,158 @@
+# # 
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Arturo Gimeno <arturo.gimeno@hpcnow.com>, Carlos Alejandro Garcia, Erica Bianco
+# License::     GPL-v3.0
+# #
+
+easyblock = 'PythonBundle'
+
+name = 'azure-cli'
+version = '2.47.0'
+
+homepage = 'https://learn.microsoft.com/en-us/cli/azure/'
+description = '''The Azure command-line interface (Azure CLI) is a set of commands used to create
+ and manage Azure resources. The Azure CLI is available across Azure services and is designed to
+ get you working quickly with Azure, with an emphasis on automation.'''
+
+versionsuffix = '-Python-%(pyver)s'
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+builddependencies = [
+    ('binutils', '2.36.1'),
+    ('CMake', '3.20.1'),
+]
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('PyYAML', '5.4.1'),
+    ('typing-extensions', '4.4.0'),
+]
+
+exts_default_options = {
+    'source_urls': [PYPI_SOURCE],
+    'use_pip': True,
+    'download_dep_fail': True,
+}
```

`PYPI_SOURCE` is the default source URL when using `PythonBundle`, and `download_dep_fail` is enabled by default.

So this whole block can be replaced with:

```suggestion
use_pip = True

---

hi @boegel, we made the requested changes and updated the pr a while ago, we would like to know if something else is required, thanks!